### PR TITLE
ci: Add basic check/fmt/clippy tests.

### PR DESCRIPTION
**Pull request by alexmv**

### CODE REVIEW

**Comment by alexmv** [comment]
@andersk: It looks like 34703d779d1c3a314e056cdf0c8a0a35e9dd3012 needs the `beta` toolchain, so I've set the CI to use that.  I think it makes sense to shift to the `stable` toolchain once `beta` gradates to that, [which will be June 30](https://forge.rust-lang.org/)?

**Comment by andersk** [comment]
#4 should fix it for `stable`.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: Swatinem/rust-cache@v1
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
```
Comment (andersk) [inline .github/workflows/ci.yaml:19]: I think this wants `args: --workspace` to check `fake-tornado` too.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - run: rustup component add clippy
+      - uses: Swatinem/rust-cache@v1
+      - uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-targets -- -D warnings
```
Comment (andersk) [inline .github/workflows/ci.yaml]: And `--workspace` here too.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    name: Rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - run: rustup component add rustfmt
```
Comment (andersk) [inline .github/workflows/ci.yaml]: Components can also be added [declaratively](https://github.com/actions-rs/toolchain#components).

**Pull request merged**